### PR TITLE
Skip test temporarily until headingLevel is added in engine (issue 41…

### PR DESCRIPTION
**Pull request by victorgalo**
The purpose of this PR is to temporarily skip one integration test that is blocking the changes indicated below:

(This change adds a new property in Semantics widget that would take an integer corresponding to the heading levels defined by the ARIA heading role. This is necessary in order to get proper accessibility and usability in a website for users who rely on screen readers and other assistive technologies.)

Issue fixed by this PR:
https://github.com/flutter/flutter/issues/97894

Engine part:
https://github.com/flutter/engine/pull/41435 

Framework part:
https://github.com/flutter/flutter/pull/125771

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Review by chunhtai — COMMENTED** [review]
There is also a similar test in custom_painter you need to skip

**Comment by Piinks** [comment]
(triage) Hi @victorgalo are you planning on returning to this change? @chunhtai mentioned one more test case, and it looks like there is a merge conflict that need resolving.

**Comment by victorgalo** [comment]
> custom_painter

Hi @Piinks sure!, I will. Sorry for taking so long to deal with this.

**Comment by victorgalo** [comment]
> There is also a similar test in custom_painter you need to skip

Hi @chunhtai, sorry to ask this but which test?, I tried to find a similar conflicting test but couldn't find any.

**Comment by chunhtai** [comment]
I think it is one of these test https://github.com/flutter/flutter/blob/bf5d2b875f784278398ee5b1eaa01b6466cb4315/packages/flutter/test/widgets/custom_painter_test.dart#L414

I think the engine smoke test would have failed the test, if not, I think it should be fine then

**Comment by victorgalo** [comment]
> I think it is one of these test
> 
> https://github.com/flutter/flutter/blob/bf5d2b875f784278398ee5b1eaa01b6466cb4315/packages/flutter/test/widgets/custom_painter_test.dart#L414
> 
> I think the engine smoke test would have failed the test, if not, I think it should be fine then

Hi @chunhtai, oh ok, but I think neither of those will fail, we can leave them. Thank you!

File: packages/flutter/test/semantics/semantics_update_test.dart [hunk 1]
```diff
@@ -84,7 +84,9 @@ void main() {
 
     SemanticsUpdateBuilderSpy.observations.clear();
     handle.dispose();
-  });
+  },
+    skip: true, // [intended] Skipped temporarily until headingLevel is added in engine (see issue 41435).
+  );
```
Comment (chunhtai) [inline packages/flutter/test/semantics/semantics_update_test.dart:89]:
```suggestion
  }, skip: true); // https://github.com/flutter/flutter/issues/97894
```

File: packages/flutter/test/semantics/semantics_update_test.dart [hunk 2]
```diff
@@ -166,7 +168,9 @@ void main() {
...
 }
```
Comment (chunhtai) [inline packages/flutter/test/semantics/semantics_update_test.dart:174]:
```suggestion
  }, skip: true); // https://github.com/flutter/flutter/issues/97894
}
```

**Review by chunhtai — APPROVED** [review]
LGTM

**Comment by victorgalo** [comment]
Hi @yjbanov, can you take a look to this please?, I need your approval to get this landed, and this PR is currently blocking these other PR's:
https://github.com/flutter/engine/pull/41435
https://github.com/flutter/flutter/pull/125771

Thank you!

**Comment by kevmoo** [comment]
Please look at the suggestions @victorgalo !

**Comment by FrancescoAbbondo** [comment]
Hello @yjbanov , can you give me a date (even indicative) for the release of this feature into Flutter? When the PR will be approved? Thanks a lot

**Comment by kevmoo** [comment]
@FrancescoAbbondo – please see the suggestions!!!

**Comment by victorgalo** [comment]
> Please look at the suggestions @victorgalo !

stupid me, sorry for overlooking the suggestions! now they are added.

**Review by kevmoo — APPROVED** [review]

**Pull request merged**